source/collaborate/chat-with-copilot.rst

```diff
@@ -84,6 +84,35 @@ To summarize unread Mattermost channels:
 .. image:: ../images/Unread-Channel-Messages-Summarization-Updated-4-3.gif
   :alt: Quickly summarize new messages, find action opens, and seek out unanswered questions with your Copilot.
 
+Summarize Zoom meetings in Mattermost
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/ent-only.rst
+  :start-after: :nosearch:
+
+.. note::
+
+  The Zoom plugin must be :doc:`enabled and configured </integration/zoom-interoperability>` by a Mattermost system admin and Zoom cloud recordings and transcripts must be enabled before you can summarize Zoom meetings.
+
+If the Zoom plugin is enabled and configured, subscribe a Mattermost channel to a Zoom meeting (``/zoom subscribe [meeting ID]``) and record the meeting. Once the recording and transcription are available, they are automatically shared back to the channel.
```

Comment:
There is other way of having summaries, that is by using regular meetings. If you create a meeting using through the zoom button in the Mattermost interface, and record it to the cloud, it will have exactly the same behavior, the `/zoom subscribe` is not strictly needed. The `/zoom subscribe` is for the scheduled meetings that we use for things like R&D.

Reply:
Thanks for pointing this out, I modified the docs to reflect.